src/relayer-provider/v2/RelayerV2Provider_input-proof.test.ts

```diff
@@ -86,6 +89,9 @@ const consoleLogSpy = jest
 const describeIfFetchMock =
   TEST_CONFIG.type === 'fetch-mock' ? describe : describe.skip;
 
+const itIfFetchMock = TEST_CONFIG.type === 'fetch-mock' ? it : it.skip;
+const itIfFetch = TEST_CONFIG.type === 'fetch-mock' ? it.skip : it;
```

Comment:
```suggestion
const [itIfFetchMock, itIfFetch] = TEST_CONFIG.type === 'fetch-mock' ? [it, it.skip] : [it.skip, it];
```

Or a destructured object to avoid unnamed vars, but it's clear enough with an array imo